nevergrad/benchmark/plotting.py

```diff
@@ -74,7 +74,7 @@ def _make_sorted_winrates_df(victories: pd.DataFrame) -> pd.DataFrame:
     algorithms comes first.
     """
     assert all(x == y for x, y in zip(victories.index, victories.columns))
-    winrates = victories / (victories + victories.T)
+    winrates = victories / (victories + victories.T).max(axis=1)
```

Comment:
we count the victories normalized over the normal number of matches, not over the number of matches we managed to play - so that algorithms which can not play in all cases (e.g. not dealing with parallelism) are penalized.

Reply:
Please update the docstring to make this clear

Reply:
Done.